app.json

```diff
@@ -37,6 +37,10 @@
       }
     },
     "android": {
+      "adaptiveIcon": {
+        "foregroundImage": "./assets/adaptive-icon.png",
+        "backgroundColor": "#0F243F"
```

Comment:
just setting the background color was not enough? did you try?

Reply:
I haven't tried it, but I don't think it will be enough, because the icon we are currently using doesn't have a safe zone. If we don't use a different icon for `foregroundImage`, the icon will look like below.

<img width="247" alt="image" src="https://github.com/user-attachments/assets/645aad93-0501-4050-8605-f18251dcd943">

Though Int/Dev may not be the right place to test this. Because there is a difference between the icon developed for municipalities and the icon developed for int/dev.

Municipalities' icons: have background color
Int/Dev icon: transparent.

I just tried it with a municipality's app and it works.

<img width="255" alt="image" src="https://github.com/user-attachments/assets/af2824c4-0a0f-46f2-82e2-ee1e342a9979">

But it still doesn't mean it will work for every municipality. For some municipalities the icons may need to be updated.

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/commit/d7441f20580fb412d7f0b85c47ff2c1c79ca1880

Reply:
all right, i understand fully now. but this means, that we should keep the foreground image as it will be necessary for most of the customers!
you could make the int/dev logo a bit smaller in the visible area to not get the edges cut i guess. give it a try 🤞

Reply:
Adjust the dimensions as in the documentation and the icon looks much more accurate.

<img width="221" alt="image" src="https://github.com/user-attachments/assets/b742ce87-a8b9-47ca-be1f-1fc39420812d">

Reply:
OK. I've added foregroundImage again. After this change, I thought we needed it too.

https://github.com/smart-village-solutions/smart-village-app-app/commit/093cb036be76d83412d729b45a527ea906c85df2

Reply:
whuuu nice!